package.json

```diff
@@ -16,6 +17,8 @@
   "devDependencies": {
     "@gulp-sourcemaps/sources-content": "^1.0.0",
     "babel-core": "^6.26.3",
+    "chokidar-cli": "^2.1.0",
+    "concurrently": "^5.2.0",
```

Comment:
IMO we should put these in the individual polyfills too. Though, I think the repo is already in a bad state where some dev dependencies are explicitly listed in the individual polyfills' packages and some aren't.

Reply:
Hmm, yeah I wasn't sure which pattern to follow. Here's a comment from one of the maintainers of Lerna suggesting putting shared `devDependencies` only at the top level: https://github.com/lerna/lerna/issues/258#issuecomment-229237660

I guess advantages of being only in the top-level are:
- Faster and smaller install.
- Keeps versions in sync.

Disadvantages:
- Can't run the command when you are cd'd into the specific package unless you manually set your path. (But maybe this reflecting the fact that you're supposed to run all your commands from the top-level using `--scope` when needed?).

Sound right? Other pros/cons? Would seem to favor putting more stuff in `devDependencies`?

There's also the `lerna link convert` command, which I hadn't looked at before: https://github.com/lerna/lerna#common-devdependencies. It makes everything a `devDependency` of the top-level, and adds `file:../` dependencies. I tried it out... but I'm not sure it's doing the right thing, it seems to do weird things, like removing all of the inter-package dependencies from `webcomponentsjs` (not even `file:` deps).

Reply:
I could probably be swayed to convert this repo to have all dev dependencies at the top, so if you want to leave them here that works for me. Though, it kind of feels off because the dependencies aren't actually at their most specific point, but I guess it's not as if we're going to work on these packages in isolation anymore.

I didn't know about `lerna link convert` - after running it I can see what you're saying about it doing weird stuff. I think the removal of all the inter-package dev dependencies makes sense from the perspective of wanting to put all dev dependencies at the top but what confuses me is that it added all of the packages as regular dependencies of the top level package:

```
+  "dependencies": {
+    "@webcomponents/custom-elements": "file:packages/custom-elements",
+    "@webcomponents/html-imports": "file:packages/html-imports",
+    "@webcomponents/shady-css-scoped-element": "file:packages/shady-css-scoped-element",
+    "@webcomponents/shadycss": "file:packages/shadycss",
+    "@webcomponents/shadydom": "file:packages/shadydom",
+    "@webcomponents/template": "file:packages/template",
+    "@webcomponents/url": "file:packages/url",
+    "@webcomponents/webcomponentsjs": "file:packages/webcomponentsjs",
+    "tests": "file:packages/tests"
   }
```

Those were never dependencies of that package, so I'm not sure why it thinks they should be there? It also added them as dev dependencies (expected) but with `file:` refs that look broken as well as not attempting to link to the local webcomponentsjs package:

```
   "devDependencies": {
     "@gulp-sourcemaps/sources-content": "^1.0.0",
+    "@webcomponents/custom-elements": "file:../custom-elements",
+    "@webcomponents/html-imports": "file:../html-imports",
+    "@webcomponents/shadycss": "file:../shadycss",
+    "@webcomponents/shadydom": "file:../shadydom",
+    "@webcomponents/template": "file:../template",
+    "@webcomponents/url": "file:../url",
+    "@webcomponents/webcomponents-platform": "^1.0.0",
+    "@webcomponents/webcomponentsjs": "^2.4.2",
     "babel-core": "^6.26.3",
```

Reply:
Ok, I'll leave it as-is, and we can look into making things more consistent and do a bit more research on the best approach later.